json_to_table/src/table.rs

```diff
@@ -335,14 +335,14 @@ mod json_to_table {
 
                 set_table_style(&mut table, config);
 
-                table.with(Width::increase(width.unwrap_or(0)));
+                table.with(Width::increase(width.unwrap_or_default()));
                 table.with(SetBottomChars(
                     used_splits,
                     table
                         .get_config()
                         .get_borders()
                         .top_intersection
-                        .unwrap_or(' '),
+                        .unwrap_or_default(),
```

Comment:
Is ' ' actually a default?
I was always thinking it's '\0'

Interesting if so.

Reply:
I stand corrected- the default char is `\x00`.

I'll revert this change